components/SiteFooter.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <v-footer fixed app dark color="#8ea1b9">
+    <v-container fluid>
+      <v-col>
+        <v-row>
+          <v-col>
+            {{ $t("footer.siteTitle", [new Date().getFullYear()]) }}
+          </v-col>
+          <v-col id="donate">
+            <a href="https://ko-fi.com/theyokohamalife" target="_blank">
+              {{ $t("footer.donate") }}</a
+            >
+          </v-col>
+          <v-col id="github">
+            <a
+              href="https://github.com/ourjapanlife/findadoc-frontend"
+              target="_blank"
+            >
+              {{ $t("footer.github") }}</a
```

Comment:
We could use a github icon here!

Reply:
Yeah it might be fun to try it. :octocat: